packages/create/templates/demo/package.template.json

```diff
@@ -2,6 +2,7 @@
 	"name": "~TODO~",
 	"version": "0.0.1",
 	"type": "module",
+	"private": true,
```

Comment:
it's pretty subjective, but I usually put `private` right after `name` in my projects because it's so important to know if something will be published or not and you really don't want to miss whether or not it's there and accidentally publish something